src/help/help.go

```diff
@@ -64,6 +68,103 @@ func help(topic string) string {
 	if f, present := m[topic]; present {
 		return helpFromBuildRule(f.FuncDef)
 	}
+
+	// Check plugins
+	if pluginHelp := helpForPlugin(topic); pluginHelp != "" {
+		return pluginHelp
+	}
+
+	return ""
+}
+
+func helpForPlugin(topic string) string {
+	config, err := core.ReadDefaultConfigFiles(nil)
+	if err != nil {
+		panic("Failed to read config")
+	}
+	if _, ok := config.Plugin[topic]; ok {
+		message := fmt.Sprintf("${BOLD_BLUE}%v${RESET} is a plugin defined in the .plzconfig file.", topic)
+		if val, ok := config.Plugin[topic].ExtraValues["subrepo"]; ok {
+			subrepo := val[0]
+			buildLabel := core.BuildLabel{PackageName: "", Name: subrepo, Subrepo: subrepo}
```

Comment:
Doesn't this cause a parse error because `///{subrepo}//:{subrepo}` doesn't exist? I think `name` should be the psudo-label `:all`?